main.js

```diff
@@ -0,0 +1,120 @@
+import createMobileMenu from './mobile_menu.js';
+import implementScrollUp from './scroll_up.js';
+
+/// ///////////////feature  speackers  section dynamic implementation ///////////
+
+const speackersInfo = [
+  {
+    name1: 'yohai benkler',
+    name2: 'Yochai Benkler',
+    img: './img/speaker_01.png',
```

Comment:
- [ ] add a little bit of margin to the images so the realy follow the behance design.. great work
your design 👇🏾 
![Screenshot (280)_LI](https://user-images.githubusercontent.com/65298048/182485847-78b34130-38ca-437c-acb8-194068e2ccdb.jpg)
behance design 👇🏾 
![Screenshot (281)_LI](https://user-images.githubusercontent.com/65298048/182485934-4f9de4ac-4b24-4998-955a-eb2c7c4a6edc.jpg)